cves/CVE-2012-0022.yml

```diff
@@ -120,9 +126,11 @@ discovered:
     If there is no evidence as to how this vulnerability was found, then you
     may leave the entries blank except for "answer", BUT please write down
     where you looked in "answer".
-  answer:
-  date:
-  automated:
+  answer: According to http://tomcat.apache.org/security-5.html, this bug was
+    found thorugh analysis of a hash collision vulnerability
```

Comment:
Maybe expand on how the hash collision vulnerability was related a little bit more. Even if it was not related at all, it still adds interesting context to the vulnerability.

Reply:
Added more description on how the two were related.